plugin/nvim-treesitter.lua

```diff
@@ -54,11 +54,13 @@ local highlights = {
   TSFloat = { link = "Float", default = true },
 
   TSFunction = { link = "Function", default = true },
+  TSFunctionCall = { link = "TSFunction", default = true },
   TSFuncBuiltin = { link = "Special", default = true },
   TSFuncMacro = { link = "Macro", default = true },
   TSParameter = { link = "Identifier", default = true },
   TSParameterReference = { link = "TSParameter", default = true },
   TSMethod = { link = "Function", default = true },
+  TSMethodCall = { link = "Function", default = true },
```

Comment:
not map to `TSMethod` to not change existing configurations?

Reply:
Good catch! I missed that one...